packages/next/test/fixtures/32-custom-install-command/package.json

```diff
@@ -3,8 +3,8 @@
     "build": "next build"
   },
   "dependencies": {
-    "next": "canary",
-    "react": "latest",
-    "react-dom": "latest"
+    "next": "13.0.4",
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
```

Comment:
Lets keep this as `canary` and `latest` so its clear the version doesnt need to be pinned.